.gometalinter.json

```diff
@@ -1,6 +1,6 @@
 {
     "Vendor": true,
-    "Exclude": ["should have comment"],
+    "Enable": ["vet", "gocyclo", "golint", "ineffassign", "misspell", "deadcode", "unconvert", "gofmt"],
```

Comment:
Could we skip the *.pb.go file for lint?

Reply:
Thx for reminding me. I'll do it.

Reply:
That's so cool!